programs/test/benchmark.c

```diff
@@ -302,7 +348,30 @@ int main( int argc, char *argv[] )
 
         for( i = 1; i < argc; i++ )
         {
-            if( strcmp( argv[i], "md4" ) == 0 )
+            if( ( q = strchr( argv[i], '=' ) ) != NULL )
```

Comment:
@AndrzejKurek , not sure I understand this logic with parsing '=' and setting \0 to *q++. Could you add a comment ?

Reply:
OK, git the logic, we are guessing that only "buffer_size" param will have "=" inside ? 
Why do we need it? It seems it is simpler to perform line 354 without lines 351-353, and use strncmp. this way we can use "=" also in other places in the future

Reply:
This exact same approach is used in other Mbed TLS programs, like here: https://github.com/PelionIoT/pelion-crypto/blob/d70746033489accee0b4ca3d04d9918df19c37ee/programs/ssl/ssl_client2.c#L1149
I used it to be consistent with other programs, and it can be used to use other parameters with "=" in the future as well :)

Reply:
OK, let's be consistent :)